packages/mux-player/src/index.ts

```diff
@@ -733,20 +735,41 @@ class MuxPlayerElement extends VideoApiElement {
   }
 
   /**
-   * Return the storyboard URL when a playback ID is provided,
+   * Return the storyboard-src attribute URL
+   */
+  get storyboardSrc() {
+    return this.getAttribute(PlayerAttributes.STORYBOARD_SRC) ?? '';
+  }
+
+  /**
+   * Set the storyboard-src attribute URL
+   */
+  set storyboardSrc(src: string) {
```

Comment:
question(blocking): Per prior comment, re: `undefined`. If `undefined` is a valid value, update type to `src: string | undefined`